client/src/components/profile/components/Camper.tsx

```diff
@@ -129,7 +129,7 @@ function Camper({
           <br />
           <p className='text-center yearsTopContributor'>
             <FontAwesomeIcon icon={faAward} />{' '}
-            <Link to={'/top-contributors'}>{t('profile.contributor')}</Link>
+            <Link {t('links:top-contributors')}={'/top-contributors'}>{t('profile.contributor')}</Link>
```

Comment:
```suggestion
            <Link to={t('links:top-contributors')}>{t('profile.contributor')}</Link>
```

Reply:
Hey, I made the changes as you had suggested, can you please have a look now.